pytorch_lightning/profiler/simple.py

```diff
@@ -73,11 +76,12 @@ def _make_report(self) -> Tuple[list, float]:
         total_duration = time.monotonic() - self.start_time
         report = [[a, d, 100.0 * np.sum(d) / total_duration] for a, d in self.recorded_durations.items()]
         report.sort(key=lambda x: x[2], reverse=True)
-        return report, total_duration
+        total_calls = sum(len(x[1]) for x in report)
+        return report, total_calls, total_duration
 
     def summary(self) -> str:
         sep = os.linesep
-        output_string = ""
+        output_string = f"{sep}"
```

Comment:
```suggestion
        output_string = ""
```

This shouldn't add an initial newline, that's the responsibility of the previous print to stdout

Reply:
we have certain warnings that come up during a run, and it looks a little ugly though during the display, so thought it might be good to add a new line to present the summary clearly.

Reply:
That would be a problem with the warnings, not the profiler summary.